src/mod/applications/mod_conference/conference_member.c

```
@@ -1652,7 +1652,7 @@ switch_status_t conference_member_say(conference_member_t *member, char *text, u
 	}
 
 	if (zstr(tts_engine) || zstr(tts_voice)) {
-		switch_log_printf(SWITCH_CHANNEL_LOG, SWITCH_LOG_ERROR, "Missing TTS engine or voice\n");
+		switch_log_printf(SWITCH_CHANNEL_LOG, SWITCH_LOG_WARNING, "Missing TTS engine or voice\n");
 		status = SWITCH_STATUS_FALSE;                                                                                                                       
 		goto end;
```

The code returns `SWITCH_STATUS_FALSE` and jumps to `end` when TTS engine or voice is missing. This suggests it's an error condition that prevents normal operation. Changing to WARNING may mask a real error that should be monitored. Consider whether this is truly just a configuration preference or an actual failure condition that should remain as ERROR.